src/lib/index.ts

```diff
@@ -421,6 +420,37 @@ export class SourceFileConverter {
               );
               partTypes.push('attribute');
               continue;
+            } else if (name.startsWith('@')) {
+              // Convert event binding into a jsaction.
+              // TODO: this functionality should be extracted into some sort of
+              // plugin or other architectural separation.
+              const handlerExpression = expressions[partTypes.length];
```

Comment:
I think this is where you're going to need to add a part type for property expressions. If you have a property expression before an event expression, they'll get misaligned I think.

Reply:
Did you foresee a reason for tracking the part type here?